src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
         // test your code here
+        DriverService driverService = (DriverService) injector
+                .getInstance(DriverService.class);
+        Driver peter = new Driver("Peter","1234567");
+        System.out.println("Create driver with name=Peter, licence number=1234567: "
+                + driverService.create(peter));;
+        Driver bob = new Driver("Bob","98765432");
+        System.out.println("Create driver with name=Bob, licence number=98765432: "
+                + driverService.create(bob));;
+        System.out.println("Get driver peter: " + driverService
+                .get(peter.getId()).toString());;
```

Comment:
toString() method will be called automatically inside System.out.println
```suggestion
                .get(peter.getId()));;
```